docs/reference/slm/apis/slm-delete.asciidoc

```diff
@@ -0,0 +1,76 @@
+[[slm-api-delete-policy]]
+=== Delete snapshot lifecycle policy API
+++++
+<titleabbrev>Delete policy</titleabbrev>
```

Comment:
I think the [API reference template](https://raw.githubusercontent.com/elastic/docs/master/shared/api-ref-ex.asciidoc) just removes the "API" from the end.
```suggestion
<titleabbrev>Delete snapshot lifecycle policy</titleabbrev>
```

I'm not entirely opposed to this, but there are several other APIs that could get the same abbreviated title. If you decide to stick with this, do you mind updating the reference template with some comments?